tasks/managed/update-trusted-tasks/README.md

```diff
@@ -0,0 +1,12 @@
+# update-trusted-tasks
+Tekton task to update trusted-tasks list bundles.
+Once a task is released as a bundle, the list of trusted-tasks should be 
+updated with the new task.
+The OCI artifact containing the trusted-tasks list named `acceptable-data-bundles` and should reside in the same org in the registry.
+If it is already in place it will be used as an input and append to it, else a new artifact will be created
```

Comment:
```suggestion
If it is already in place, it will be used as an input to which the results will be appended, else a new artifact will be created.
```